set-env.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+export PS1="(devbox)$PS1"
```

Comment:
My command line shows `(devbox)` twice. Could that be because of this line?

<img width="148" alt="image" src="https://github.com/user-attachments/assets/09440525-1fe6-4dcb-b69e-c7e160a1e653">

Reply:
Did you enter `devbox shell` in another `devbox shell`? Kind of like `inception`?

Reply:
I am running it as described [below](https://github.com/AngelaE/ts-mountebank/pull/36#issuecomment-2366431881), with devbox starting automatically. If I remove that `(devbox)` here it only shows devbox once in my prompt.